packages/design-system-docs/src/styles/components/_ReactPropDocs.scss

```diff
@@ -0,0 +1,65 @@
+// Responsive docs table
+// TODO: Deprecate after the release of React Table Component. This include removing from m.gov child ds.
+
+.docs_table--container {
```

Comment:
Why is this being added back? I thought it wasnt needed anymore if we are using the React table

Reply:
I put this back temporarily as child ds will fail if it's removed. This will then be removed after child ds .scss has been updated to refer to core for the styles.

Reply:
sorry im not sure if i understand. why would the child ds fail? isnt the child ds using the react table for the react props?

Reply:
The child ds docs/src/index.scss file make references to core ds for the "ReactPropDocs". I believe we can remove "ReactPropDocs" after we release the pr for chld ds where it references core ds for "index.scss" instead of the individual reusable files.

Reply:
It doesn’t affect the child ds to remove this file, as it will be removed in the next version. the child ds is using version 2.0.0 which has the file